pkg/plugins/resources/postgres/store.go

```diff
@@ -273,6 +280,29 @@ func rowToItem(resources model.ResourceList, rows *sql.Rows) (model.Resource, er
 	return item, nil
 }
 
+func (r *postgresResourceStore) countRows(resource, mesh string) (int, error) {
+	statement := `SELECT COUNT(*) as count FROM resources WHERE type=$1`
+	var statementArgs []interface{}
+	statementArgs = append(statementArgs, resource)
+	argsIndex := 1
+	if mesh != "" {
+		argsIndex++
+		statement += fmt.Sprintf(" AND mesh=$%d", argsIndex)
+		statementArgs = append(statementArgs, mesh)
+	}
+	statement += " GROUP BY name,mesh ORDER BY name, mesh"
```

Comment:
Can you help me understand why do we need group by and order here?
I'm a little bit rusty with SQL. I always used count + group by like this
```
SELECT type, count(*) GROUP BY type
```
but when the `count(*)` is the only column, what is the role of group by?

Reply:
Hmm, actually this whole line can go away I think. In any case, if I want to have ORDER BY, then I get `failed to execute query: SELECT COUNT(*) as count FROM resources WHERE type=$1 ORDER BY name, mesh: pq: column "resources.name" must appear in the GROUP BY clause or be used in an aggregate function`. I think there was a suggestion to actually add this ORDER BY.

Reply:
I also think that if you remove group by + order by it should work fine.

Reply:
Removed it, looks fine to me.